vspace/src/main/resources/config_reference.properties

```diff
@@ -0,0 +1,33 @@
+reference_type_BLOG_POST=Blog Post
+reference_type_BOOK=Book
+reference_type_BOOK_SECTION=Book Section
+reference_type_CONFERENCE_PAPER=Conference Paper
+reference_type_JOURNAL_ARTICLE=Journal Article
+reference_type_PATENT=Patent
+reference_type_PODCAST=Podcast
+reference_type_PRESENTATION=Presentation
+reference_type_PROJECT_DELIVERABLE=Project Deliverable
+reference_type_PROJECT_MILESTONE=Project Milestone
+reference_type_SOFTWARE_DOCUMENTATION=Software Documentation
+reference_type_THESIS=Thesis
+reference_type_TECHNICAL_NOTE=Technical Note
+reference_type_WEB_PAGE=Web Page
+reference_type_WORKING_PAPER=Working Paper
+reference_type_OTHER=Other
+
+BLOG_POST=blogPost
+BOOK=book
+BOOK_SECTION=bookSection
+CONFERENCE_PAPER=conferencePaper
+JOURNAL_ARTICLE=journalArticle
+PATENT=patent
+PODCAST=podcast
+PRESENTATION=presentation
+PROJECT_DELIVERABLE=projectDeliverable
+PROJECT_MILESTONE=projectMilestone
+SOFTWARE_DOCUMENTATION=softwareDocumentation
+THESIS=thesis
+TECHNICAL_NOTE=technicalNote
+WEB_PAGE=webPage
+WORKING_PAPER=workingPaper
+OTHER=other
```

Comment:
let's prefix these properties with something that explains what they are for (like the once above).